### PR TITLE
add lightbox

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: tempelhof/assets/js/app.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// lightbox needs the jQuery object in global scope
+require('./jquery-fix.js')
 const collapse = require('./collapse.js')
 const menu = require('./menu')
-const calendar = require('events/static/events/calendar.js')
```
Comment (xi) [inline tempelhof/assets/js/app.js]: Why was this removed?

**Comment by xi** [comment]
The opening animation is slightly off (FOUC), but good enough for me.

File: tempelhof/assets/js/app.js (hunk 1 above)
Comment (vellip) [inline tempelhof/assets/js/app.js]: Good question, most likely a mistake.
